src/main.cpp

```diff
@@ -55,6 +55,42 @@ extern void qt_set_sequence_auto_mnemonic ( bool bEnable );
 
 // Implementation **************************************************************
 
+// NOTE: To prevent memory leaks any pointers assigned with new
+//       should be declared here and checked and deleted in reverse order of creation in cleanup().
+//       cleanup() should be called before any exit() !
```

Comment:
I'm wondering if it was harder to forget calling cleanup if we had a generic `CleanExit(int)` function?

Reply:
> I'm wondering if it was harder to forget calling cleanup if we had a generic `CleanExit(int)` function?

Yes I thought about that too, but then I could not use it for the normal exit.
I would prefer using an exception instead of using exit() even more...
(Note that calling exit() instead of normally returning from main with a return code may skip some cleanup in windows itself, and maybe it's the same on other OS's, so this increases the chance on memory leaks.)